specification/maps/data-plane/Common/preview/1.0/common.json

```diff
@@ -1,712 +1,712 @@
-      "headers": {
-        "Location": {
-          "type": "string",
-          "description": "New URL to check for the results of the long running process."
-        }
+    "parameters": {
+      "ClientId": {
+        "name": "x-ms-client-id",
+        "description": "Specifies which account is intended for usage in conjunction with the Azure AD security model.  It represents a unique ID for the Azure Maps account and can be retrieved from the Azure Maps management  plane Account API. To use Azure AD security in Azure Maps see the following [articles](https://aka.ms/amauthdetails) for guidance.",
+        "type": "string",
+        "in": "header",
+        "x-ms-parameter-location": "client"
+      },
+      "Language": {
+        "name": "language",
+        "in": "query",
+        "description": "Language in which search results should be returned. Should be one of supported IETF language tags, case insensitive. When data in specified language is not available for a specific field, default language is used.\n\nPlease refer to [Supported Languages](https://docs.microsoft.com/en-us/azure/azure-maps/supported-languages) for details.",
+        "type": "string",
+        "x-ms-parameter-location": "method"
+      },
+      "ResponseFormat": {
+        "name": "format",
+        "description": "Desired format of the response. Only `json` format is supported.",
+        "type": "string",
+        "in": "path",
+        "required": true,
+        "enum": [
+          "json"
+        ],
+        "x-ms-enum": {
+          "name": "ResponseFormat",
+          "modelAsString": true,
+          "values": [
+            {
+              "value": "json",
+              "description": "[The JavaScript Object Notation Data Interchange Format](https://tools.ietf.org/html/rfc8259)"
+            }
+          ]
+        },
+        "x-ms-parameter-location": "method"
+      },
+      "TextFormat": {
+        "name": "format",
+        "description": "Desired format of the response. Value can be either _json_ or _xml_.",
+        "type": "string",
+        "in": "path",
+        "required": true,
+        "enum": [
+          "json",
+          "xml"
+        ],
+        "x-ms-enum": {
+          "name": "TextFormat",
+          "modelAsString": true,
+          "values": [
+            {
+              "value": "json",
+              "description": "[The JavaScript Object Notation Data Interchange Format](https://tools.ietf.org/html/rfc8259)"
+            },
+            {
+              "value": "xml",
+              "description": "[The Extensible Markup Language](https://www.w3.org/TR/xml/)"
+            }
+          ]
+        },
+        "x-ms-parameter-location": "method"
+      },
+      "GeographicResourceLocation": {
+        "name": "geography",
+        "description": "This parameter specifies where the Azure Maps Creator resource is located.  Valid values are us and eu.",
```

Comment:
It seems this is inaccurate, as more resources than just Creator use this parm.

Reply:
Those resources (spatial/alias/data/...) using parameters are all under `Creator` now. (Please ignore `data-plane/Microsoft.Maps`. This is outdated)